src/server/reply_send.c

```diff
@@ -175,10 +187,25 @@ dis_reply_write(int sfds, struct batch_request *preq)
 {
 	int rc;
 	struct batch_reply *preply = &preq->rq_reply;
+#ifndef WIN32
+      struct sigaction act, oact;
+#endif
+
+	pbs_tcp_timeout = PBS_DIS_TCP_TIMEOUT_REPLY;
```

Comment:
perhaps we store the current value of pbs_tcp_timeout in a local variable and reset back to it at the end of this function. This would prevent any overriding change if there is a dialogue going on - not all ifl requests might be setting th timeout consistently.